Description: make tst_qqmlapplicationengine pass when JIT is not available
 When the platform does not have JIT, a warning about this is printed
 into the process stderr, so the test fails.
Origin: upstream, https://code.qt.io/cgit/qt/qtdeclarative.git/commit/?id=650b5db06b057abe
Last-Update: 2016-10-14

--- a/tests/auto/qml/qqmlapplicationengine/tst_qqmlapplicationengine.cpp
+++ b/tests/auto/qml/qqmlapplicationengine/tst_qqmlapplicationengine.cpp
@@ -111,7 +111,7 @@
     test_stderr_target.replace('\n', QByteArray("\r\n"));
 #endif
     QCOMPARE(test_stdout, QByteArray(""));
-    QCOMPARE(QString(test_stderr), QString(test_stderr_target));
+    QVERIFY(QString(test_stderr).endsWith(QString(test_stderr_target)));
     delete testProcess;
     QDir::setCurrent(srcDir);
 #else // !QT_NO_PROCESS
